src/containers/ChartView/index.tsx

```diff
@@ -68,17 +96,38 @@ const ChartView = () => {
         availableDates,
     ]);
 
+    const handleChartDataChange = (
+        event: React.MouseEvent<HTMLElement>, //needs to be here for correct type check
+        newChartType: 'cumulative' | 'nDaysAverage',
+    ) => {
+        if (!newChartType) return;
+
+        setChartType(newChartType);
+    };
+
     return (
         <ChartContainer>
-            <Typography variant="body1">
-                Total confirmed cases:{' '}
+            <Typography
+                variant="body1"
+                sx={{ width: '100%', textAlign: 'center' }}
+            >
+                {chartTypes[chartType].title}
                 <strong>
                     {selectedCountry
                         ? getCountryName(selectedCountry.name)
                         : `Worldwide`}
                 </strong>
             </Typography>
-
+            <ToggleButtonGroup
+                color="primary"
+                value={chartType}
+                exclusive
+                onChange={handleChartDataChange}
+                aria-label="Platform"
```

Comment:
This should be changed to something more specific to the function of those buttons